src/VisualStudio/IntegrationTests/VisualStudioIntegrationTests.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\build\Targets\VSL.Settings.targets" />
+  <PropertyGroup>
+    <ProjectGuid>{E5A55C16-A5B9-4874-9043-A5266DC02F58}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Roslyn.VisualStudio.Integration.UnitTests</RootNamespace>
+    <AssemblyName>Roslyn.VisualStudio.Integration.UnitTests</AssemblyName>
+    <PlatformTarget>x86</PlatformTarget>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <Nonshipping>true</Nonshipping>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="App.config" />
+    <None Include="project.json" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="CSharp\CSharpBuild.cs" />
+    <Compile Include="CSharp\CSharpInteractiveDemo.cs" />
+    <Compile Include="SharedIntegrationHost.cs" />
```

Comment:
Why is this defined here versus defined in the library project itself? I'm on the fence either way, curious what your thoughts are.